ExtIO_sddc/ExtIO_sddc.vcxproj

```diff
@@ -254,6 +254,8 @@
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
+    <ClInclude Include="pffft\fmv.h" />
```

Comment:
please update CMakeFile.txt to add pf_mixer.cpp to SRCS list

Reply:
and add include folder as well. I can help you want to.

Reply:
in my ExtIO_sddc/ExtIO_sddc.vcxproj I see:
 </ItemDefinitionGroup>
  <ItemGroup>
    <ClInclude Include="pffft\fmv.h" />

I have only  CMakeLists.txt no CMakeFile.txt ?

Reply:
my bad. it should be CMakeLists.txt.

Reply:
Apply the following patch to CMakeLists.txt:

diff --git a/ExtIO_sddc/CMakeLists.txt b/ExtIO_sddc/CMakeLists.txt
index f008757..e500608 100644
--- a/ExtIO_sddc/CMakeLists.txt
+++ b/ExtIO_sddc/CMakeLists.txt
@@ -3,9 +3,9 @@ project(ExtIO_RX888)

 set(CMAKE_CXX_FLAGS "-O2 /std:c++17 /EHsc /W3")

-file(GLOB SRC "*.cpp" "radio/*.cpp" "CyAPI/*.cpp" extio.def resource.rc manifest.xml)
+file(GLOB SRC "*.cpp" "radio/*.cpp" "CyAPI/*.cpp" "pffft/*.cpp" extio.def resource.rc manifest.xml)

-include_directories("." "CyAPI/" "FFTW3/")
+include_directories("." "CyAPI/" "FFTW3/" "pffft/")

 set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -D_DEBUG")